docs/en_US/deploy-locally.md

```diff
@@ -0,0 +1,207 @@
+# Deploy ob-operator locally
+
+## Hardware requirements
+
+* CPU: 4 Core available
+* Memory: 10 GB available (A Single-node OceanBase cluster takes 8 ~ 9 GB memory)
+* Disk space: 120 GB available
+
+## Software requirements
+
+You should install following two software before we start. Installation instructions could be found on their sites. You can get minkube by brew install minikube if you develop on macOS.
+* [kubernetes/minikube](https://github.com/kubernetes/minikube): Run kubernetes locally
+* [Docker desktop](https://www.docker.com/products/docker-desktop/): Manage container development locally
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl): Kubernetes CLI (optional if you prefer minikube kubectl)
+
+## Setup steps
+
+### 1. Configure Docker Resources
+
+Open Docker desktop - Settings - Resources, set `CPU limit`, `Memory limit` and `Virtual disk limit` to match the hardware requirements.
+
+![Docker Limits](../img/docker-limit.png)
+
+### 2. Start minikube cluster
+
+In order to make sure that the hardware limit are enforced to VM in minikube, we set configurations of minikube again by:
+
+```shell
+minikube config set cpus 4
+minikube config set disk-size 120GB
+minikube config set memory 10GB
+```
+
+Here we go! Just type `minikube start` and enter, we could see it starts fetching necessary images. After it says initialization is finished, we could type `kubectl get pods -A` (or `minikube kubectl -- get pods -A`) to see all pods in minikube cluster.
+
+Tips: Perform `minikube dashboard` to open kubernetes dashboard, everything in the cluster could be more clear.
+
+### 3. Install cert-manager
```

Comment:
the process to deploy the components in kubernetes cluster is duplicated
I'd like to suggest to change this document only focused on setting up the kubernetes environment

Reply:
The existing `deploy` documentation reveals only installation instructions of ob-operator. No clusters and tenants are involved, while they are essential for developers to see what they would behave.